tests/test_compression_lr_multiplier.py

```diff
@@ -0,0 +1,268 @@
+
+
+def get_params_grouped_by_algorithms(model: nn.Module) -> Dict[str, Iterable[nn.Parameter]]:
+    cls_name_to_params = {}
+    modules = model.modules()
+    for module in modules:
+        params = module.parameters(recurse=False)
+        full_cls_name = '.'.join([module.__class__.__module__, module.__class__.__name__])
+        if full_cls_name not in cls_name_to_params:
+            cls_name_to_params[full_cls_name] = []
+        cls_name_to_params[full_cls_name].extend(params)
+
+    algo_name_to_params = {'regular': []}
+    for cls_name, params in cls_name_to_params.items():
+        params = [param for param in params if param.requires_grad]
+        for algo_name, algo_path in ALGO_NAME_TO_PATH_MAP.items():
+            if algo_path in cls_name:
+                if algo_name not in algo_name_to_params:
+                    algo_name_to_params[algo_name] = []
+                algo_name_to_params[algo_name].extend(params)
+                break
+        else:
+            algo_name_to_params['regular'].extend(params)
+
+    return algo_name_to_params
+
+
+def get_params_after_train_steps(config: NNCFConfig, num_steps: int = 1) -> Dict[str, Iterable[nn.Parameter]]:
+    with set_torch_seed():
+        model = LeNet()
+        model, compression_ctrl = create_compressed_model_and_algo_for_test(model, config)
+
+        for param in model.parameters():
+            if param.requires_grad:
+                nn.init.normal_(param)
+        optimizer = SGD(model.parameters(), lr=0.1)
+
+        for i in range(num_steps):
+            optimizer.zero_grad()
+            x = torch.rand(config['input_info']['sample_size'])
+            y = model(x)
+            loss = (y ** 2).sum()
+
+            loss.backward()
+            optimizer.step()
+
+    return get_params_grouped_by_algorithms(model)
+
+
+def test_if_algorithms_add_params(ref_and_target_configs: Tuple[NNCFConfig, NNCFConfig]):
+    base_config, config_with_multiplier = ref_and_target_configs
+
+    algo_to_params = get_params_after_train_steps(config_with_multiplier)
+    algo_names = get_multipliers_from_config(base_config).keys()
+
+    assert sorted(algo_to_params.keys()) == sorted(list(algo_names) + ['regular'])
+
+
+def test_how_multipliers_affect_grads(ref_and_target_configs: Tuple[NNCFConfig, NNCFConfig]):
+    base_config, config_with_multiplier = ref_and_target_configs
+
+    ref_params = get_params_after_train_steps(base_config)
+    target_params = get_params_after_train_steps(config_with_multiplier)
+    multipliers = get_multipliers_from_config(config_with_multiplier)
+    multipliers['regular'] = 1
+
+    for algo in multipliers:
+        ref_grads = get_grads(ref_params[algo])
+        ref_grads = [multipliers[algo] * grad for grad in ref_grads]
+        target_grads = get_grads(target_params[algo])
+
+        check_equal(ref_grads, target_grads)
+
+
+def test_how_multipliers_affect_params_change(ref_and_target_configs: Tuple[NNCFConfig, NNCFConfig]):
```

Comment:
A good test name describes what is being tested. This is not always possible due to the constraint on the length of the function name, but in this case, the test case could be named `test_setting_multipliers_affects_model_parameters`

Reply:
@vanyalzr we need you as a tie-breaker here. Which of the two variants of the test naming sounds better to you?

Reply:
`test_setting_multipliers_affects_model_parameters` sounds better to me

Reply:
Done.